src/dashboard/dashboard.controller.ts

```diff
@@ -29,18 +22,25 @@ export class DashboardController {
 
   @ApiResponse({
     status: HttpStatus.OK,
-    type: () => GetUserAmountDto,
-    description: 'Total registered users',
+    type: () => GetReceiptAmountDto,
+    description: 'Total recirpt amount',
   })
   @ApiOperation({
-    summary: 'Get total registered users',
+    summary: 'Get total recirpt amount',
+  })
+  @ApiQuery({
+    name: 'date',
+    type: Number,
+    required: true,
+    description: 'Date (UnixTimeStamp)',
```

Comment:
(Optional) please also provide that the timestamp muse be in seconds

Reply:
discuss w/ golf he said no

Reply:
frontend not gonna be dumb if it false its their false